### PR TITLE
New loadouts

### PR DESCRIPTION
**Pull request by Freddo3000**
I've for whatever reason been making a pretty large amount of loadouts over the course of the last week, and figured why not try and add them to the modpack.

Added loadouts:

- Russian Ground Forces (2018, EMR Ratnik)
- US Army Rangers (1990~1995, M81)
- US Army Rangers (2000-2005, M81)
- US Marines (1985, M81)

### CODE REVIEW

**Comment by beaar** [comment]
Did you test these?

**Comment by beaar** [comment]
Feel free to make more exotic loadouts 😉

**Comment by Freddo3000** [comment]
@beaar Tested them in editor and using the TMF autotest, all issues have been resolved. I'll try and make more exotic loadouts in the future.

**Comment by beaar** [comment]
Do you want me to merge this now?

**Comment by Freddo3000** [comment]
Yes I would say that you can merge it now.

File: addons/tm_tmf_loadouts/CfgLoadouts.hpp [hunk 1]
```diff
@@ -80,6 +80,18 @@ class CfgLoadouts
 		category = "USA";
 		#include "loadouts\us_army_airborne_ocp.hpp"
 	};
+	class US_ARMY_RANGERS_1990_M81
+	{
+		displayName = "US Army Rangers (1990~1995, M81)";
+		category = "USA";
+		#include "loadouts\us_army_rangers_1990_m81"
```
Comment (beaar) [inline addons/tm_tmf_loadouts/CfgLoadouts.hpp]: missing .hpp suffix

File: addons/tm_tmf_loadouts/CfgLoadouts.hpp [hunk 2]
```diff
@@ -80,6 +80,18 @@ class CfgLoadouts
...
+	};
+	class US_ARMY_RANGERS_2000_M81
+	{
+		displayName = "US Army Rangers (2000-2005, M81)";
+		category = "USA";
+		#include "loadouts\us_army_rangers_2000_m81"
```
Comment (beaar) [inline addons/tm_tmf_loadouts/CfgLoadouts.hpp]: missing .hpp suffix

File: addons/tm_tmf_loadouts/CfgLoadouts.hpp [hunk 3]
```diff
@@ -335,7 +359,18 @@ class CfgLoadouts
...
+	{
+		displayName = "Horizon Island Defence Force (1990)";
+		category = "Armaverse Factions";
+		#include "loadouts\hidf_1990.hpp"
+	}
+	class HIDF_2000
+	{
+		displayName = "Horizon Island Defence Force (2000)";
+		category = "Armaverse Factions";
+		#include "loadouts\hidf_2000"
```
Comment (beaar) [inline addons/tm_tmf_loadouts/CfgLoadouts.hpp]: missing .hpp suffix

File: addons/tm_tmf_loadouts/CfgLoadouts.hpp [hunk 4]
```diff
@@ -172,6 +190,12 @@ class CfgLoadouts
 		category = "Russia";
 		#include "loadouts\ru_msv_emr.hpp"
 	};
+	class RUS_MSV_EMR_RATNIK
+	{
+		displayName = "RU Ground Forces (2018, EMR Ratnik)";
```
Comment (beaar) [inline addons/tm_tmf_loadouts/CfgLoadouts.hpp]: You can specify that the loadout uses AK-12s, e.g. `(2018, EMR, AK-12s)`. You don't have to, but I like to do it for loadouts with non-standard guns

**Comment by Freddo3000** [comment]
I've applied fixes and tested them in game using the build.exe method. From my point of view, it should be ready to be merged.

File: addons/tm_tmf_loadouts/CfgLoadouts.hpp [hunk 5]
```diff
@@ -335,7 +359,24 @@ class CfgLoadouts
...
+	};
+	class HIDF_2000
+	{
+		displayName = "Horizon Island Defence Force (2000)";
+		category = "Armaverse Factions";
+		#include "loadouts\hidf_2000.hpp"
+	};
+	class SYNDIKAT_1990
+	{
+		displayName = "Syndikat (1990-, Bandits)";
```
Comment (cptnnick) [inline addons/tm_tmf_loadouts/CfgLoadouts.hpp]: typo

File: addons/tm_tmf_loadouts/CfgLoadouts.hpp [hunk 6]
```diff
@@ -80,6 +80,18 @@ class CfgLoadouts
 		category = "USA";
 		#include "loadouts\us_army_airborne_ocp.hpp"
 	};
+	class US_ARMY_RANGERS_1990_M81
+	{
+		displayName = "US Army Rangers (1990~1995, M81)";
```
Comment (cptnnick) [inline addons/tm_tmf_loadouts/CfgLoadouts.hpp]: inconsistent ~

File: addons/tm_tmf_loadouts/CfgLoadouts.hpp (hunk 5 above)
Comment (Freddo3000) [reply addons/tm_tmf_loadouts/CfgLoadouts.hpp]: Not a typo, they're called Syndikat with a K.

Comment (cptnnick) [reply addons/tm_tmf_loadouts/CfgLoadouts.hpp]: i meant the - after 1990, shoudl have clarified, sorry. Also what does Bandits indicate?

Comment (Freddo3000) [reply addons/tm_tmf_loadouts/CfgLoadouts.hpp]: the - indicates from 1990 and forwards. As for Bandits, the vanilla faction is split up between paramilitary and bandits.

**Comment by beaar** [comment]
Thanks @Freddo3000

**Pull request merged**